### PR TITLE
Feature: Camera and Math

### LINKED ISSUE #36: Implement Camera Support in Render Pipeline
There should be some method for the game script to set/control the cameras.  Camera class works fine, but the render pipeline doesn't know anything about the camera.

### LINKED ISSUE #39: Quaternion.fromEulerAngles is Broken
Quaternion.fromEulerAngles( 0, 0, 0 ) results in a quaternion representing a 180 degree rotation about the Y axis.

### LINKED ISSUE #40: Rotations are Currently in Radians
These should probably be in degrees.

### LINKED ISSUE #56: Integrate gl3n Math library

### PR DESCRIPTION
**Pull request by wozniakty**
Camera and Math should be working.  Please test this one carefully, as there were hours of headache put into it, and I would be lying if I said I was confident it all worked perfect.

Closes #56.
Closes #40.
Closes #39.
Closes #36.

### CODE REVIEW

**Comment by dislogical** [comment]
@PxlBuzzard was able to create a basic game on the `InputBinding` branch, which uses this. I think the only downside we found was that you have to manually update each object's matrix every time you move it.

tldr; :a:

**Comment by ericchristenson** [comment]
So our coordinate system is right handed now?
Edit: I just care because of that wiki page I made.

**Comment by brooklynlittell** [comment]
Yup!

**Comment by brooklynlittell** [comment]
:+1:

**Comment by tmr9209** [comment]
Looks good to me.
